arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-preparing-node-config.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: 'How to Configure a Node Using the Orbit SDK'
-sidebar_label: 'Node Config Generation'
+title: "How to configure your Orbit chain's node using the Orbit SDK"
+sidebar_label: "Configure your chain's node"
```

Comment:
Sentence casing, title patterns ([ref](https://www.notion.so/arbitrum/How-to-775951f3d84c476cb6cba296a0a6f671?pvs=4))